app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
- Dùng gem config cho https://www.railstutorial.org/ vào Settings rồi gọi ra ở đây
- Dùng link_to thay cho thẻ a
- 18n toàn bộ file này